include/deal.II/lac/linear_operator.h

```diff
@@ -1506,15 +1498,12 @@ template <
   typename OperatorExemplar,
   typename Matrix,
   typename = typename std::enable_if<
-    !std::is_lvalue_reference<OperatorExemplar>::value>::type>
+    !std::is_lvalue_reference<OperatorExemplar>::value>::type,
+  typename = typename std::enable_if<
+    !std::is_same<OperatorExemplar,
+                  LinearOperator<Range, Domain, Payload>>::value>::type>
```

Comment:
@masterleinad Thanks. I missed that one.

Reply:
I would probably not have found this without declaring the function as `deleted`.